zproject/dev_settings.py

```diff
@@ -40,13 +40,15 @@
 
 # Uncomment extra backends if you want to test with them.  Note that
 # for Google and GitHub auth you'll need to do some pre-setup.
-AUTHENTICATION_BACKENDS = (
+auth_backends = (
     'zproject.backends.DevAuthBackend',
     'zproject.backends.EmailAuthBackend',
     'zproject.backends.GitHubAuthBackend',
     'zproject.backends.GoogleAuthBackend',
     # 'zproject.backends.AzureADAuthBackend',
 )
+# Allows for typing with tuples of variable length
+AUTHENTICATION_BACKENDS = cast(Tuple[Any], auth_backends)
```

Comment:
I think this change is wrong; `AUTHENTICATION_BACKENDS` should just be switched to a list[str] everywhere (that's been supported for ages; I think we just never migrated).

Reply:
This (along with other `dev_settings` changes, and the `timezone` commit) are meant to be removed.  They were actually changed in the mypy upgrade PR.  The change was in 77068f3e1e1bf56f48c7dc076ff7463783bb1796.

Reply:
I'll make the changes you are suggesting here. :)